src/ui/public/paginated_table/paginated_table.html

```diff
@@ -33,5 +33,5 @@
   <!-- auto-inserted by the paginate directive... -->
   <!-- <paginate-controls></paginate-controls> -->
   <div class="pagination-container" ng-transclude></div>
-
+  
```

Comment:
Stray white space, you might configure your editor to strip trailing white space on save